pyproject.toml

```diff
@@ -100,77 +100,55 @@ line-length = 88
 
 [tool.ruff]
 line-length = 88
+target-version = "py39"
+extend-exclude = ["doc/*", "benchmarks/*", "versioneer.py", "geopandas/_version.py"]
+
+[tool.ruff.lint]
 select = [
     # pyflakes
     "F",
     # pycodestyle
     "E",
     "W",
-    # flake8-2020
-    "YTT",
+    # pyupgrade
+    # "UP",
     # flake8-bugbear
     "B",
-    # flake8-quotes
-    "Q",
     # flake8-debugger
     "T10",
-    # flake8-gettext
-    "INT",
+    # flake8-simplify
+    # "SIM",
     # pylint
     "PLC",
     "PLE",
     "PLR",
     "PLW",
     # misc lints
     "PIE",
-    # flake8-pyi
-    "PYI",
```

Comment:
Removed because we don't have any pyi stub files (and I think in the future if we add more type annotations, we will do that inline, given we are pure python)